lib/watir/locators.rb

```diff
@@ -26,23 +26,23 @@ module Watir
   module Locators
     module ClassHelpers
       def locator_class
-        class_from_string("#{Watir.locator_namespace}::#{element_class_name}::Locator") ||
+        class_from_string("#{query_scope.locator_namespace}::#{element_class_name}::Locator") ||
```

Comment:
@akostadinov we don't have `#locator_namespace` defined for `Watir::Element`, which is a valid query_scope instance. I think for this we want to hard code this to `browser`, and keep query_scope as an instance variable.